src/wazuh_modules/vulnerability_detector/wm_vuln_detector.h

```diff
@@ -132,6 +132,20 @@ typedef enum vu_task_type_t {
     VU_TASK_TYPES_SIZE // This should be the last constant
 } vu_task_type_t;
 
+typedef enum vu_interval_feed_update_t {
```

Comment:
We should discuss with the team if creating a new array is the best and maintainable way of working with the flags.

Another option would be to have an array with all the `vu_feed` constants even when some of them will never be used.

Reply:
I initially think to implement it as you describe, I wrongly took the other approach that is not easy to maintain and it is not intuitive that you should modify this enum if the vu_feed enum changes.